inc/options/class-wpseo-option-wpseo.php

```diff
@@ -337,6 +338,12 @@ protected function validate_option( $dirty, $clean, $old ) {
 
 					break;
 
+				case 'semrush_country_code':
+					if ( isset( $dirty[ $key ] ) ) {
+						$clean[ $key ] = $dirty[ $key ];
+					}
+					break;
```

Comment:
Not my area of expertise but I assume this doesn't need a real validation because these are hardcoded values in the countries select that users can't change. correct?
Besides that, it's worth considering to avoid duplicated code and maybe add this case to the existing `previous_version` case which uses the same code.

Reply:
Also: I see there's a method to validate the value from the REST API and also tests for that. Shouldn't we validate the value here as well instead of accepting any value?

Reply:
The code above is needed to at least save the option. It's true that users would usually select the code from the list but this doesn't prevent people to write code to set the option.
I agree that we should validate the values here as well. We should open another issue to centralize the country code list so it can be accessed by this validation, the REST API one (to provide a meaningful error), and the country selector.